python/ray/rllib/models/model.py

```diff
@@ -65,6 +65,7 @@ def __init__(self,
         self.options = options
         self.scope = tf.get_variable_scope()
         self.session = tf.get_default_session()
+        self.beholder_tensors = []
```

Comment:
Create a new class in wgym that derives from model, move this and the method below there, and make all our model extend this new class.

Reply:
You can then remove watch_tensor method from wgym/models/common.py